### DISCUSSION TITLE
My project was corrupted

### DISCUSSION THREAD

**Discussion by henokteixeira** [community/community#67716] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

The day before yesterday I started a project in an organization created here on GitHub. I assigned several issues and assigned them to the organization's convenient repositories. 

It turns out that yesterday, when accessing the project, I saw that some of the last stories I had created were not in the project, as if they had not been saved; In addition, the issues that I had assigned to a repository had been disconnected from the project, leaving only a few draft issues in the project, which I tried to delete and presented an error. 
Instantly, I saw that GitHub services were unstable.

A few minutes later I accessed the project again and there were no more issues; it was clean. 

And this morning, when I tried to access the project, I couldn't. It gives an error "this project failed to load".

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by duo755** [comment]
I have the exact same issue. I'm so sad 😢 . I'm really hoping there might be a way to roll it back to before the issue happened yesterday.

**Reply by duo755** [reply]
They are investigating  https://github.com/orgs/community/discussions/67636#discussioncomment-7060312
